src/Zendesk.hs

```diff
@@ -144,6 +145,69 @@ runZendeskMain = do
             printWarning
             tickets <- runApp (listTickets Assigned) cfg
             printTicketCountMessage tickets (cfgEmail cfg)
+        ShowStats2 -> do
+            putTextLn $ "Testing - Statistics associated with: "
+               <> cfgEmail cfg
+            tickets <- runApp (listTickets Assigned) cfg
+            statsExample
+            -- How many tickets are open
+            putTextLn $ "There are " <> show (length $ filter ((== "open") . ticketStatus) tickets) <> " open tickets."
+            -- How many tickets are closed
+            putTextLn $ "There are " <> show (length $ filter ((== "closed") . ticketStatus) tickets) <> " closed tickets."
+            -- How many open tickets have attachments
+            let openTickets = filter (\ticket -> ticketStatus ticket == "closed") tickets --TODO: CHANGE TO OPEN (Only closed shows results as there are no open tickets according to the analyzer)
+            runApp (showTicketsWithAttachments openTickets) cfg
+            putTextLn $ "End of statistics."
+
+showTicketsWithAttachments :: [TicketInfo] -> App ()
+showTicketsWithAttachments tickets = do
+  liftIO $ putTextLn "Scanning tickets for attachments"
```

Comment:
Follow indentation rules (4 spaces) and style guide used here - https://github.com/serokell/serokell-util/blob/master/serokell-style.md